git-scripts.gemspec

```diff
@@ -20,8 +20,8 @@ Gem::Specification.new do |s|
    s.add_dependency 'json', '~> 1.8.0'
    s.add_dependency 'multi_json', '= 1.8.0'
    s.add_dependency 'highline', '= 1.6.19'
-   s.add_dependency 'faraday', '= 0.8.8'
-   s.add_dependency 'faraday_middleware', '= 0.9.0'
+   s.add_dependency 'faraday'
+   s.add_dependency 'faraday_middleware'
```

Comment:
I highly doubt that octokit v3 doesn't specify its dependencies correctly. I bet it'll work if you drop this and only specify the dependencies *we* need.